workflow.md

```diff
@@ -58,7 +58,7 @@ information.
 
 After review by another team member, the teammate merging the PR should merge
 `master` into the Pull request's branch *without* the `--no-ff` option, and then
-the PR branch *with* the `--no-ff` option enabled.
+the PR branch into `master`.
```

Comment:
You did something wrong here. Merge into master should be done with `no-ff`, but for merges into feature branch it's irrelevant (because ff can't happen anyway).